### ISSUE TITLE
Maxwell (v1.8.0) does not handle references to table "rds_heartbeat2" from Aurora. 

### ISSUE THREAD

**Issue by brettbevers** [zendesk/maxwell#632]
I am testing Maxwell with RDS Aurora in preparation for migrating our Maxwell stream over from RDS mysql. Most operations seem to be working fine. Maxwell can connect to Aurora, setup the maxwell database, receive/process/publish bin logs. However, Maxwell eventually crashes with the following error:
```
19:08:06,883 DEBUG MaxwellKafkaProducer -    BinlogPosition[mysql-bin-changelog.001404:24641311]
19:08:06,883 DEBUG MaxwellKafkaProducer - 
19:08:06,887 DEBUG TransportImpl - disconnected from host.rds.amazonaws.com:3306
java.lang.RuntimeException: Couldn't find table rds_heartbeat2
        at com.zendesk.maxwell.replication.TableCache.processEvent(TableCache.java:29)
        at com.zendesk.maxwell.replication.TableCache.processEvent(TableCache.java:42)
        at com.zendesk.maxwell.replication.MaxwellReplicator.getTransactionRows(MaxwellReplicator.java:250)
        at com.zendesk.maxwell.replication.MaxwellReplicator.getRow(MaxwellReplicator.java:346)
        at com.zendesk.maxwell.replication.AbstractReplicator.work(AbstractReplicator.java:127)
        at com.zendesk.maxwell.util.RunLoopProcess.runLoop(RunLoopProcess.java:35)
        at com.zendesk.maxwell.Maxwell.start(Maxwell.java:181)
        at com.zendesk.maxwell.Maxwell.main(Maxwell.java:202)
```
Either the table `rds_heartbeat2` does not exist in database `mysql` on Aurora or it is not visible. In any case, Maxwell does not capture `rds_heartbeat2` in its internal representation of the databases, and it fails when attempting to process bin logs regarding that table.

Is there some configuration that I need to change on either the Maxwell or Aurora side?

**Comment by brettbevers** [comment]
It appears that adding `blacklist_tables=rds_heartbeat2` to my config file has gotten me past the error. Is this the recommended way to handle this issue?

**Comment by osheroff** [comment]
No, someone should get together a patch here.  There's already a bit of code that works around this issue in alibaba rds, I think we can just add this table to the "system blacklist".

**Comment by timbertson** [comment]
see #633

**Comment by timbertson** [comment]
fixed in 1.10.0 (just released)
